### PR TITLE
calculator-component

### PR DESCRIPTION
**Pull request by Obimbo07**
This pull request introduces the following calculator component.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+          >
+            /
+          </button>
+          <button
+            type="button"
+            className="items item5 light-bg"
+            onClick={() => handleClick('7')}
+          >
+            7
+          </button>
```
Comment (rloterh) [inline src/components/Calculator.js]: - [ ] Nice job adding the Calculator component, however, let's ensure that it has one child, as per the project requirements. You can create a separate component, for instance, a Button component and then use it as a child component inside in the Calculator component.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Obimbo, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁 🚦 ⛪ 🚜 🏃 🌴 🏫 🌳 🏘️

## Highlights 👍  
- Calculator design matches given template!
- Good job so far on component creation!
- Good ReadMe!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import Result from './Result';
+
+const Calculator = () => {
+  const handleClick = (buttonName) => (`${buttonName}clicked`);
```
Comment (LionRouge1) [inline src/components/Calculator.js:5]: [OPTIONAL]
- [ ] I noticed this `handleClick` function, but it appears to be doing nothing. As a best practice and in line with the YAGNI, I would recommend that you remove it from your code, you can always add it back at the time that you need it.

**Review by LionRouge1 — APPROVED** [review]
Hi @Obimbo07, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**